app/services/ipam/base.rb

```diff
@@ -17,6 +17,18 @@ def initialize(opts = {})
       normalize_mac!
     end
 
+    def subnet_range
+      @subnet_range ||= begin
+        subnet_range = IPAddr.new("#{subnet.network}/#{subnet.mask}", subnet.family).to_range
+        # exclude first element - network
```

Comment:
I haven't changed that, this was the original code. This is used in IPv6 as well, so I assume yes. @timogoebel ?